### PR TITLE
Adding public getter to ObligationId #IDENTITY-4889 #12

### PR DESCRIPTION
**Pull request by ManZzup**
Added the public getter to Obligation Id field in Obligation class
Issue tracked at https://wso2.org/jira/browse/IDENTITY-4889

### CODE REVIEW

**Comment by cdwijayarathna** [comment]
Don't we need to change balana version to 1.1.0-SNAPSHOT since we are adding new public functions here?

**Comment by ManZzup** [comment]
Didn't know that, I have to change it in the pom i guess?

File: modules/balana-core/src/main/java/org/wso2/balana/xacml3/Obligation.java [hunk 1]
```diff
@@ -128,6 +128,12 @@ public void encode(StringBuilder builder){
         return assignments;
     }
 
+    /**
+     * Returns the obligation id of the obligation object
+     * @return the obligation Id
+     */
+    public URI getObligationId(){ return obligationId; }
```
Comment (mefarazath) [inline modules/balana-core/src/main/java/org/wso2/balana/xacml3/Obligation.java]: Check formatting.

**Comment by pulasthi7** [comment]
Hi @ManZzup , Can you fix the above and update the PR?

**Comment by mefarazath** [comment]
@ManZzup appreciate if you could address the same for xacml2 ObligationId as well. Refer https://github.com/wso2/balana/issues/12

**Comment by ManZzup** [comment]
Hi, yes I can fix for xacml2 as well, in the initial PR should it come in 2 lines? @mefarazath

**Comment by mefarazath** [comment]
```
 public URI getObligationId() { 
     return obligationId;
 }
```
need a space between ) and { as above :smile:

**Comment by ManZzup** [comment]
@mefarazath ohh fixed that xD and added for xacml2 as well

**Comment by cdwijayarathna** [comment]
Shall we update the PR title? I just shocked for a moment after seeing that 1613 files changed to add a getter method :)

**Comment by darshanasbg** [comment]
Resent the fix with: https://github.com/wso2/balana/pull/54

Hence closing this PR

**Pull request closed without merging**